test/library/packages/ConcurrentMap/PREDIFF

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sed -E "s/0x[0-9a-f]*/prediffed/" <$2 >$2.predifftmp
+grep prediffed $2.predifftmp | uniq -c >$2
+rm $2.predifftmp
```

Comment:
I recommend adding bad files for these futures. It'll help us notice if there's a new leak.

This prediff is has no use, otherwise.

Reply:
I've removed the prediff, since even after doing `sort | uniq -c` there was still non-determinism.